lib/vm/src/lib.rs

```diff
@@ -17,5 +17,5 @@ pub use crate::calls::{
 };
 pub use crate::compatability::check_api_compatibility;
 pub use crate::instance::Instance;
-pub use crate::memory::read_memory;
+pub use crate::memory::{read_memory, read_region};
```

Comment:
I wonder if this re-export makes sense. Why would a user of the VM need to call `read_region` directly?

Reply:
You're right. No need to export this.

Reply:
Thanks. Removed it before the renaming now: c1098ebd216bda6f300fb888bf06d6720045c6bd